app/src/main/res/layout/activity_reset_password__page.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Make the reset password page look more like this
Make the confirm button red
Make the hint be "Enter Code Number" instead of the text so that you can type in the in box and the hint text disappears.
![image](https://user-images.githubusercontent.com/38587739/67540521-a3d90b00-f6b3-11e9-8470-6d68a8d93951.png)